server/sonar-server/src/main/java/org/sonar/server/debt/DebtModelBackup.java

```diff
@@ -58,10 +59,11 @@
   private final RuleDefinitionsLoader defLoader;
   private final System2 system2;
   private final UserSession userSession;
+  private final RuleIndexer ruleIndexer;
 
   public DebtModelBackup(DbClient dbClient, RuleOperations ruleOperations,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00107)